clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferWgsUbamV1Key.scala

```diff
@@ -6,3 +6,8 @@ case class TransferWgsUbamV1Key(flowcellBarcode: String,
                                 lane: Int,
                                 libraryName: String,
                                 location: Location)
+    extends TransferKey {
+
+  def getUrlPath: String =
+    s"$flowcellBarcode/$lane/$libraryName/$location"
```

Comment:
I almost feel like the location should be the first path segment similar to gvcfs. That way we are consistent with increasing specificity as you move down the segments. location -> flowcell -> lane -> library

Reply:
That would be OK with me, doing it now would be better than later. What do you think, @tbl3rd and @danxmoran ?

Reply:
+1

Reply:
Generally: I think we should review this PR for maintainability regressions (and easy improvements) only and approve it otherwise.

Specifically: I see $location as different in kind from $barcode, $lane, and $library. I like moving it first because it is going to be shared across all of our indexes. And another issue just occurred to me: Some indexes will have multiple paths, but only one $location, so ...